### PR TITLE
Retries

### PR DESCRIPTION
**Pull request by volkerstampa**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Set up Python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
```
Comment (ahartel) [inline .github/workflows/integration.yml:17]: why was that change necessary?

Comment (volkerstampa) [reply .github/workflows/integration.yml:17]: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows
and
https://github.com/actions/setup-python/releases/tag/v4.3.0
(there is a comment in the git-commit for this change)

**Comment by volkerstampa** [comment]
Hmmm ... not sure. I thought this is a (friendly) way of pushing ppl to the new API. But I could also add it to the deprecated one.

**Comment by benbrandt** [comment]
@volkerstampa I'd say we can put the new defaults in the old client, to at least help with the default case, but let's leave the opportunity to set it only on the new one

**Review by benbrandt — APPROVED** [review]
